Platon/backend/app/workspace_system/helpers.py

```diff
@@ -51,16 +51,74 @@ def add_workspace_contribution(workspace_id, user_id):
 	db.session.commit()
 
 
-def get_workspace_skills_text(workspace_id):
-	# Skill list of the requested workspace is retrieved from the database.
-	requested_workspace_skills = WorkspaceSkill.query.filter_by(workspace_id=workspace_id)
-	return [Skills.query.filter_by(id=workspace_skill.skill_id).first().name for workspace_skill in requested_workspace_skills]
+def get_workspace_skills_json(workspace_id):
+    # Skill list of the requested workspace is retrieved from the database.
+    requested_workspace_skills = WorkspaceSkill.query.filter_by(workspace_id=workspace_id)
 
+    # After the list is retrieved, the name of each skill of that workspace is retrieved from the database
+    # and gets appended to "workspace_skill_names"
+    workspace_skill_names = list()
+    for workspace_skill in requested_workspace_skills:
+        skill_name = (Skills.query.filter_by(id=workspace_skill.skill_id).first()).name
+        workspace_skill_names.append(skill_name)
+    # The list of the names of the skills are converted to a JSON string and returned.
+    return json.dumps(workspace_skill_names)
```

Comment:
Why do you dump this list into JSON string? It makes it harder to use for the person who will use them on Frontend or Android.